packages/itensor/quri_parts/itensor/sampler.py

```diff
@@ -54,9 +54,10 @@ def create_itensor_mps_concurrent_sampler(
     """Returns a :class:`~ConcurrentSampler` that uses ITensor mps simulator
     for sampling.
 
-    For now, this function works when the executor is defined like below
-    `with ProcessPoolExecutor(max_workers=2,
-    mp_context=get_context("spawn")) as executor:`
+    For now, this function works when the executor is defined like below::
+
+        with ProcessPoolExecutor(max_workers=2, mp_context=get_context("spawn"))
+        as executor:
```

Comment:
It should be like this:
```suggestion
    with ProcessPoolExecutor(
        max_workers=2, mp_context=get_context("spawn")
    ) as executor:
        pass
```